internal/errors.proto

```diff
@@ -4,17 +4,6 @@ option go_package = "internal";
 
 import "google/protobuf/any.proto";
 
-// Error is the generic error returned from unary RPCs.
-message Error {
-	string error = 1;
-	// This is to make the error more compatible with users that expect errors to be Status objects:
-	// https://github.com/grpc/grpc/blob/master/src/proto/grpc/status/status.proto
-	// It should be the exact same message as the Error field.
-	int32 code = 2;
-	string message = 3;
-	repeated google.protobuf.Any details = 4;
-}
-
 // StreamError is a response type which is returned when
 // streaming rpc returns an error.
 message StreamError {
```

Comment:
Can we delete this too? Not sure we need it to be anything other than a `status` either.

Reply:
Sure! I'll get rid of it once I've dealt with the fallout of removing `Error`. 👍